src/llamacrawl/__init__.py

```
@@ -3,12 +3,16 @@
 import warnings
 
 try:
-    from pydantic._internal._generate_schema import UnsupportedFieldAttributeWarning
+    from pydantic._internal._generate_schema import (  # type: ignore
+        UnsupportedFieldAttributeWarning,
+    )
+
+    unsupported_field_warning: type[Warning] = UnsupportedFieldAttributeWarning
 except Exception:  # pragma: no cover - defensive
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)